packages/core/src/theme/createTheme.ts

```diff
@@ -27,7 +27,10 @@ export function createTheme() {
     const breakpoints = getSlice('breakpoints');
     if (typeof value === 'object' && breakpoints) {
       const keys = Object.keys(value);
-      return keys.some(key => breakpoints[key] !== undefined);
+      return (
+        keys.includes('default') ||
+        keys.some(key => breakpoints[key] !== undefined)
+      );
```

Comment:
@lucacacciarru  I'm pretty sure this is not needed anymore because of the other fix - but it's hard to test for me, whenever this will be merged can you please try if your problem is still reproducible even without this and in case revert it?

Reply:
Of course, I will try to work without this specific fix and revert it in case it's useless